.github/workflows/build_docker_images.yml

```diff
@@ -97,11 +97,11 @@ jobs:
           push: true
           tags: atddocker/vz-socrata-export:${{ github.ref == 'refs/heads/production' && 'production' || 'development' }}
 
-      - name: Build and push PDF maintenance image
-        if: steps.changes.outputs.process_missing_cr3_pdfs == 'true'
+      - name: Build and push CR3 PDF metadata image
+        if: steps.changes.outputs.populate_cr3_file_metadata == 'true'
         uses: docker/build-push-action@v4
         with:
           platforms: linux/amd64,linux/arm64
-          context: atd-etl/process_missing_cr3_pdfs
+          context: atd-etl/populate_cr3_file_metadata
           push: true
-          tags: atddocker/vz-process-missing-cr3-pdfs:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
+          tags: atddocker/vz-pdfs:${{ github.ref == 'refs/heads/master' && 'production' || 'latest' }}
```

Comment:
I guess these are the only pdfs in vz so its not confusing, but what do you thinking about naming the image vz-cr3-metadata-pdfs or something similar?

Reply:
Yea, I think that's a really great idea @chiaberry, I'm going to push up a change with that in it.

Reply:
Thanks again for the suggestion, here's a commit naming it `vz-cr3-metadata-pdfs`: 1d59ca78a4de338761dd9f843092d832917467bc